### PR TITLE
command-line-usage added

### PR DESCRIPTION
**Pull request by matrumz**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by mhegazy** [comment]
@matrumz can you please refresh this PR and resolve the merge conflicts?

**Comment by matrumz** [comment]
@mhegazy I'm trying to get @Dvorsky 's attention so we can resolve this issue. I believe his stuff should be overwritten so the typings match the original developer's documentation, but so far no luck communicating with him.

**Comment by mhegazy** [comment]
> @mhegazy I'm trying to get @Dvorsky 's attention so we can resolve this issue. I believe his stuff should be overwritten so the typings match the original developer's documentation, but so far no luck communicating with him.

And that is fine, just update your PR to have the definitions you think are correct. now you cannot see the difference because of the merge conflicts.

**Comment by matrumz** [comment]
@mhegazy Ah, I see. I've resolved the conflict and all Travis issues.

File: types/command-line-usage/index.d.ts [hunk 1]
```diff
@@ -1,48 +1,81 @@
 // Type definitions for command-line-usage 5.0
 // Project: https://github.com/75lb/command-line-usage#readme
-// Definitions by: Andrija Dvorski <https://github.com/Dvorsky>
+// Definitions by: matrumz <https://github.com/matrumz>
```
Comment (mhegazy) [inline types/command-line-usage/index.d.ts]: do not remove the author. just add your name to the list.

**Comment by mhegazy** [comment]
@Dvorsky can you please review this change?

**Review by weswigham — APPROVED** [review]
If someone's unhappy with some of the slight structural changes, it can always be changed again at that point in time; but the documentation and test improvements certainly make this a good PR.

**Pull request merged**